app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,10 +29,22 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * Returns all products in this application.
```

Comment:
`in this application.`은 너무 뻔한 설명이라 제외해도 좋을 것 같습니다.
```suggestion
     * Returns all the products.
```

Reply:
수정하였습니다 !!